packages/stitch/src/types.ts

```diff
@@ -70,9 +80,11 @@ export interface IStitchSchemasOptions<TContext = any> extends Omit<IExecutableS
 export type SubschemaConfigTransform = (subschemaConfig: SubschemaConfig) => SubschemaConfig;
 
 export interface TypeMergingOptions {
+  selectCanonicalTypeCandidate?: (candidates: Array<MergeTypeCandidate>) => MergeTypeCandidate;
```

Comment:
Should there be a parallel option for fields?

Meaning, in one place we allow selecting the canonical type, but we don't allow customizing the actual type merging, but for fields we allow customizing the merge itself...

Reply:
Hmm, this is roughly the same as the other mergers. Maybe the name should be changed to `typeCandidateMerger` for consistency with the others... it does pretty much the same thing. Basically what happens now is that we identify a preferred candidate for each type up front, move that to the end of the list, and then resolve descriptions, fields, input fields, enum values as normal going by the "last definition wins" policy. If there is no `typeCandidateMerger` specified, then everything matches the current default behavior.

Reply:
name updated.